### PR TITLE
116031175 display number of products in stock

### PR DESCRIPTION
**Pull request by sofiengwin**
#### What does this PR do?

It allows users to see the quantity of product in stock.
#### Description of Task to be completed?

Add a poros class that reduces the quantity of a product based on the quantity of the corresponding order item. 
#### How should this be manually tested?

By visiting a product page.
#### What are the relevant pivotal tracker stories?
#116031175

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def index
   end
 
   def show
-    @product = Product.find_by_id(params[:id])
+    @product = Product.find_by_id(params[:id]).decorate
```
Comment (makinwab) [inline app/controllers/products_controller.rb]: **Nitpick - Code Quality**
You should make use of the `find_by` method like so: 
`Product.find_by(id: params[:id]).decorate`

Comment (sofiengwin) [inline app/controllers/products_controller.rb]: Fixed this issue and all other recommendations raised during the code review session.

File: spec/features/stock_counter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "StockCounter", type: :feature, js: true do
```
Comment (makinwab) [inline spec/features/stock_counter_spec.rb]: Do we still have a `StockCounter` class?

Comment (sofiengwin) [inline spec/features/stock_counter_spec.rb]: Sorry about this. I changed the class name to "Stock" but forgot to change the test description in the spec file. I will fix this immediately.

**Pull request merged**